src/composables/FileExplorer.js

```diff
@@ -54,9 +56,13 @@ export function createFile(fileInformation, parentFolder, name) {
  * @param {number} index - Index of the part to check in the path.
  */
 export function updateFileInformation(parentNode, fileStatus, index = 0) {
-  const splitPath = fileStatus.path.split('/');
+  const splitPath = fileStatus.path.split('/').slice(1);
```

Comment:
can you add a comment to explain why you have to slice 1

Reply:
yes